code/modules/networks/computer3/computer3.dm

```diff
@@ -314,6 +318,12 @@
 
 		src.base_icon_state = src.icon_state
 
+		if(glow_in_dark_screen)
+			src.screen_image = image('icons/obj/computer_screens.dmi', src.icon_state, -1)
+			src.overlays += image(screen_image)
+			screen_image.plane = PLANE_LIGHTING
+			src.UpdateOverlays(screen_image, "screen_image")
```

Comment:
ummmmmmm this doesn't look right!!!!

Reply:
you should not be using updateoverylays with manaul overlay adding and clearing

Reply:
I resolved this by removing updateoverlays, turns out that I didn't need them at all

Reply:
i meant the opposite 😳

Reply:
! i dont know what you mean then or how to fix

Reply:
one moment